lib/libc/riscv/sys/cerror.S

```diff
@@ -33,23 +33,36 @@
  */
 
 #include <machine/asm.h>
+#include "SYS.h"
 __FBSDID("$FreeBSD$");
 
+#define FRAME_SIZE	(2 * __SIZEOF_POINTER__) /* Always multiple of 16 */
```

Comment:
I would leave these all inlined since they now can be, but otherwise this looks good.

Reply:
I quite like having symbolic names rather than inline constants. I find it a bit easier to see that those are actually always the same offsets. But I don't really mind.

Reply:
I'm fine either way.